app/src/components/domain/RiskImminentEvents/Gdacs/EventDetails/LayerDetails/index.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+    Checkbox,
+    Container,
+    TextOutput,
+} from '@ifrc-go/ui';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+
+import {
+    LAYER_CYCLONE_BUFFERS,
+    LayerOption,
+    LayerType,
+} from '#utils/domain/risk';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+export interface Props {
+    options: LayerOption;
+    value: Record<LayerType, boolean>;
+    onChange: (value: boolean, name: LayerType) => void;
```

Comment:
Let's rename these values to have more semantic names.

Reply:
what about `layer` and `onLayerChange` ??